System/Matrix.h

```diff
@@ -14,10 +14,9 @@ namespace RTE {
 	public:
 
 		SerializableOverrideMethods
-
-		float m_Rotation; //!< The angle represented, in radians.
-		bool m_Flipped[2]; //!< Whether or not this Matrix also mirrors the X axis of its invoked Vectors.
-		float m_Elements[2][2]; //!< The elements of the matrix, which represent the angle.
+		float m_Rotation; //!< The angle, represented in radians. Pi/2 points up.
+		bool m_Flipped[2]; //!< Whether or not this Matrix also mirrors the respective axes of its invoked Vectors.
+		float m_Elements[2][2]; //!< The elements of the matrix, which represent the negative of the angle. Allows matrix\vector multiplications while considering upside-down coordinate system.
```

Comment:
"Allows matrix\vector multiplications" should be "Allows matrix/vector multiplications".

Reply:
no problem. i hope it's clear though, i wasn't referring to division.

Reply:
Oh yeah, I can see how that might confusing. Maybe replace it with `and` or `or` or `and/or`?

Reply:
i'll just rewrite it for clarity.